### PR TITLE
Expand a bit the qnet documentation

### PR DESCRIPTION
**Pull request by oleg-alexandrov**
## Description

This adds a little blurb describing how to open the qnet window and what to expect. This may look obvious to seasoned users, but this was not explicitly stated, and the whole experience of understanding what this tool does and how to go about it was quite confusing and took some time to figure out. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by oleg-alexandrov** [comment]
I added a blub for jigsaw too.

File: isis/src/qisis/apps/qnet/qnet.xml [hunk 1]
```diff
@@ -30,6 +30,10 @@
     </p>
 
     <p>
+      This program should be started by typing its name in a terminal, with no arguments. 
+      This will start blank versions of the main Qnet window and the control network 
+      navigator window.
+
```
Comment (Kelvinrr) [inline isis/src/qisis/apps/qnet/qnet.xml]: Probably better to word in such a way that makes it obvious this is a GUI application. As ISIS GUI apps follow the same paradigm. 

So, start with something like: "This is an ISIS GUI application. To run it with entering the command in the terminal with no parameters to start a blank session and select your files through the navigator windows."

Comment (oleg-alexandrov) [reply isis/src/qisis/apps/qnet/qnet.xml]: I added an update to the text ([d4153ee](https://github.com/DOI-USGS/ISIS3/pull/5373/commits/d4153ee19fc5f824565775cd8b52cd422c9bb3fc))

**Review by Kelvinrr — APPROVED** [review]
doc changes do not need a changlog entry, so I think this is good.

**Pull request merged**